Src/AutoFixture/CustomizeAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Reflection;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Base class for customizing parameters in methods.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = true)]
+    public abstract class CustomizeAttribute : Attribute
```

Comment:
To be direct, I don't think this class belongs in the 'core' AutoFixture library.

Can we think of a single, realistic usage scenario where this class is useful in itself, without being tied to one of the unit testing Glue Libraries?